static/app/views/performance/utils/useModuleURL.tsx

```diff
@@ -65,6 +73,14 @@ export function useModuleURLBuilder(bare: boolean = false): URLBuilder {
         : normalizeUrl(`/organizations/${slug}/${moduleURLSegment}`);
     }
 
+    if (!isInsightsEnabled && PERFORMANCE_MODULE_BASE_URLS[moduleName]) {
+      return bare
+        ? `${insightsURL}/${PERFORMANCE_MODULE_BASE_URLS[moduleName]}`
+        : normalizeUrl(
+            `/organizations/${slug}/${insightsURL}/${PERFORMANCE_MODULE_BASE_URLS[moduleName]}`
+          );
+    }
+
```

Comment:
This is pretty confusing. Can this be expressed the same way as the AI logic above? e.g., "if the module is Resources, set the correct base URL either `/assets` or `/resources`, and then concatenate". This stuff with checking `PERFORMANCE_MODULE_BASE_URLS` is fine but it's so much more confusing than just having special logic for Resources, which is what's happening here